src/index.html

```diff
@@ -22,6 +22,355 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
```

Comment:
pls format your html according to this rule
![image](https://github.com/user-attachments/assets/2c743170-6659-4b80-8e77-17494c96e045)